build/bazel/remote/asset/v1/remote_asset.proto

```diff
@@ -201,6 +201,13 @@ message FetchBlobRequest {
   //
   // Specified qualifier names *MUST* be unique.
   repeated Qualifier qualifiers = 5;
+
+  // The digest function the server must use to compute the digest.
+  //
+  // If unset, the server SHOULD select one of MD5, MURMUR3, SHA1, SHA256,
```

Comment:
I like Mostyn's suggestion in https://github.com/bazelbuild/remote-apis/pull/273/files#r1344582116 to add a field to the response indicating what digest was actually used, as a way of differentiating servers that honour vs ignore this field.

Reply:
Added